test/python/circuit/test_controlled_gate.py

```diff
@@ -756,7 +756,7 @@ def test_open_controlled_gate(self):
         ctrl_state = 0
         cgate = base_gate.control(num_ctrl_qubits, ctrl_state=ctrl_state)
         target_mat = _compute_control_matrix(base_mat, num_ctrl_qubits, ctrl_state=ctrl_state)
-        np.set_printoptions(linewidth=200,)
+        np.set_printoptions(linewidth=200, )
```

Comment:
This is a debugging line I forgot about. Can you delete?

Reply:
Here! https://github.com/Qiskit/qiskit-terra/pull/4295